make/langtools/netbeans/nb-javac/pom-nb-javac.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
@matthiasblaesing , this file doesn't have proper license header file and gets included on Maven central. Don't you want to create a PR to add the license?